### PR TITLE
Change view -> display in Shell commands

### LINKED ISSUE #490: Standardize Shell Commands - Display versus View
Several shell commands are overlapping in meaning: 

* counter display
* field-value-counter display

but 

* task view
* job execution step view

Should we go with display for all of them?

### PR DESCRIPTION
**Pull request by ilayaperumalg**
- update task/job commands

This resolves #490

### CODE REVIEW

File: spring-cloud-dataflow-shell/src/test/java/org/springframework/cloud/dataflow/shell/command/JobCommandTemplate.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ public CommandResult jobStepExecutionProgress(long id, long jobExecutionId) {
 	 * @return the results of the shell command.
 	 */
 	public CommandResult jobStepExecutionView(long id, long jobExecutionId) {
```
Comment (cppwfs) [inline spring-cloud-dataflow-shell/src/test/java/org/springframework/cloud/dataflow/shell/command/JobCommandTemplate.java]: need to rename view to display.  Will fix on merge.

File: spring-cloud-dataflow-shell/src/test/java/org/springframework/cloud/dataflow/shell/command/JobCommandTemplate.java [hunk 2]
```diff
@@ -60,26 +60,26 @@ public CommandResult jobExecutionListByName(String jobName) {
...
 	/**
 	 * Return the results of executing the shell command:
-	 * <code>dataflow: job instance view --id 1</code> where id is the id for the job
+	 * <code>dataflow: job instance display --id 1</code> where id is the id for the job
 	 * instance requested.
 	 *
 	 * @param id the identifier for the job instance.
 	 * @return the results of the shell command.
 	 */
 	public CommandResult instanceView(long id) {
```
Comment (cppwfs) [inline spring-cloud-dataflow-shell/src/test/java/org/springframework/cloud/dataflow/shell/command/JobCommandTemplate.java]: need to rename view to display.  Will fix on merge.

File: spring-cloud-dataflow-shell/src/test/java/org/springframework/cloud/dataflow/shell/command/JobCommandTemplate.java [hunk 3]
```diff
@@ -60,26 +60,26 @@ public CommandResult jobExecutionListByName(String jobName) {
...
 	/**
 	 * Return the results of executing the shell command:
-	 * <code>dataflow: job execution view --id 1</code> where id is the id for the job
+	 * <code>dataflow: job execution display --id 1</code> where id is the id for the job
 	 * execution requested.
 	 *
 	 * @param id the identifier for the job execution.
 	 * @return the results of the shell command.
 	 */
 	public CommandResult executionView(long id) {
```
Comment (cppwfs) [inline spring-cloud-dataflow-shell/src/test/java/org/springframework/cloud/dataflow/shell/command/JobCommandTemplate.java]: need to rename view to display.  Will fix on merge.

**Comment by cppwfs** [comment]
Need to update src/main/asciidoc/tasks.adoc to replace view with display.  Will fix on merge.

**Comment by ilayaperumalg** [comment]
@cppwfs thank you for taking care of them.

**Comment by cppwfs** [comment]
@ilayaperumalg Glad I could help!   LGTM

**Comment by cppwfs** [comment]
Fixes applied.  Squashed Merged.

**Pull request closed without merging**